### ISSUE TITLE
Framez 3?

### ISSUE THREAD

**Issue by farinc** [amadornes/Framez#48]
Sorry for pressure, but do you know when framez 3 will be released?

**Comment by amadornes** [comment]
Not really. School, work and other projects are keeping me really busy. I'm trying to give all projects the same amount of time, but it's hard to keep up and not get burned out.

Hopefully in the first quarter of 2017 if I have enough free time. But I can't say anything for sure.

**Comment by Notawallplan** [comment]
Any milestones on Framez 3 yet? I know you're kinda busy, but we gots to know.
